spec/features/user_browse_items_spec.rb

```diff
@@ -0,0 +1,28 @@
+# As an unauthenticated user...
+#When I visit the home page...
+#Then I expect to see a list of available spooky items (root to items_path)
+
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.